promptsource/templates/ade_corpus_v2/Ade_corpus_v2_classification/templates.yaml

```diff
@@ -2,28 +2,28 @@ dataset: ade_corpus_v2
 subset: Ade_corpus_v2_classification
 templates:
   56bd12a8-b8ee-464e-98cc-5f586ba9f74d: !Template
-    answer_choices: Not-Related ||| Related
+    answer_choices: Not Related ||| Related
     id: 56bd12a8-b8ee-464e-98cc-5f586ba9f74d
     jinja: Is "{{text}}" related to adverse drug effect (ADE)? ||| {{answer_choices[label]}}
     metadata: !TemplateMetadata
-      choices_in_prompt: null
-      metrics: []
+      choices_in_prompt: true
+      metrics:
+      - Accuracy
       original_task: true
-    name: baseline
+    name: binary-classification
     reference: ''
   78c4ce65-dd66-46ed-878d-11f4eca5e544: !Template
-    answer_choices: Yes, it is related to adverse drug effect. ||| No, it is not related
-      to adverse drug effect.
+    answer_choices: Yes ||| No
```

Comment:
This doesn't follow the required format of responding A or B if those are used to indicate choices. More generally, these just seem like weird ways of phrasing the question. Maybe both the classification tasks should just use Yes and No as the targets.